ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -943,3 +970,13 @@ extension DrawerController: UIGestureRecognizerDelegate {
         return gestureRecognizer == resizingGestureRecognizer && otherGestureRecognizer == contentScrollView?.panGestureRecognizer
     }
 }
+
+// MARK: - DrawerController: DrawerPresentationDelegate
+
+extension DrawerController: DrawerPresentationDelegate {
+    func backgroundViewTapped(_ presentationControler: DrawerPresentationController) {
```

Comment:
I think this logic of checking for the delegate return and dismissing (or moving back) the presenting view controller should be encapsulated in a single public method.

This will ensure the DrawerPresentationDelegate is not needed and will simplify the logic on the animation .ended case.

Reply:
I have created `dismissPresentingViewController` to keep code cleaner. I am not sure how I can avoid `DrawerPresentationDelegate` creation, I'll need this protocol to propagate background view tap events.